Added RC

egid1999 opened the pull request:
Res:
WR06X2322FTL
CRCW08051R00FKEAHP
MCWW12W0R05FTL

Caps:
GRM188R60J106ME47J
GRM21BR61C226ME44L
C3216X7R1H105K160AE

egid1999 committed:
Added RC

Res:
WR06X2322FTL
CRCW08051R00FKEAHP
MCWW12W0R05FTL

Caps:
GRM188R60J106ME47J
GRM21BR61C226ME44L
C3216X7R1H105K160AE

Aleksandrovas commented:
Nė vienos iš nurodytos varžos nėra, tą galite matyti skiltyje "Files Changed" ten nurodyta, kad tik kondensatorių bylą yra pakeista.

egid1999 committed:
Update Resistors.SchLib

Aleksandrovas commented:
Vistiek nurodytų rezistorių nėra: WR06X2322FTL, CRCW08051R00FKEAHP, MCWW12W0R05FTL

egid1999 committed:
Added R, C, IC

IC:
LM8261M5/NOPB
MCP6021T-E/OT
C:
GRM1885C1H472JA01D
R:
ERJ3EKF2702V
CRCW08051K20FKEA

egid1999 committed:
Update Resistors.SchLib

Aleksandrovas committed:
Merge branch 'KTU_Lib_master' into pr/51
